CIP-0119/README.md

```diff
@@ -0,0 +1,242 @@
+
+Participants at the workshop held to debut the first draft of this CIP strongly advocated in favour of a CIP that would include features that would assist a DRep comprised of a team of individuals properly describe their endeavour to delegating Ada Holders and anyone else reading the metadata associated with their registration.
+
+This CIP has not included these features, the decision not to include these features was made in order to simplify this CIP so that it became suitable for the minimum viable level of tooling, with the expectation that further CIPs will build on it. 
+
+### Witnesses
+DRep Metadata will not follow the CIP-100 specification related to signing the metadata, the [authors property](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0100/README.md#high-level-description) can be left blank without the need for tooling providers to warn their users that the author has not been validated. Instead the author can be derived from the DRep ID associated with the registration. The need for an `authors` field will also be discarded in favour of including usernames inside of the `body` field. For the avoidance of doubt this CIP recommends that the entire authors property be left blank, and that tooling ignore it. 
+
+### Extended Vocabulary
+Like CIP-108, this CIP also extends the potential vocabulary of CIP-100's `body` property. 
+
+>**Reminder for tooling providers/builders** DRep metadata is user generated content.
+
+The following are a list of properties tooling should expect to encounter:
+
+#### `paymentAddress`
+- Optional
+
+Dreps may want to recieve tokens for a variety of reasons such as:
+- donations
+- expenses
+- any incentive program
+
+Therefore there MAY be a `paymentAddress` field in the metadata where such payments could be sent. This makes such an address public and payments to DReps transparent. 
+
+#### `username`
+- Compulsory
+- A very short freeform text field. Limited to 80 characters.
+- This MUST NOT support markdown text styling.
+- It is intended that authors will use this field for their profile name/ username.
+
+#### `picture`
+- Optional 
+- A base 64 encoded profile picture
+- This SHOULD be treated as the profile picture of the individual
+
+#### `objectives`
+- Optional
+- A freeform text field with a maximum of 1000 characters
+- A short description of what the person believes and what they want to achieve as a DRep
+
+#### `motivations`
+- Optional
+- A freeform text field with a maximum of 1000 characters
+- A short description of why they want to be a DRep, what personal and professional experiences have they had that have driven them to register 
+
+#### `qualifications`
+- Optional
+- A freeform text field with a maximum of 1000 characters 
+- A space where the registrant can to list the qualifications they hold that are relevant to their role as a DRep
+
+#### `references`
+- Compulsory
```

Comment:
```suggestion
- Compulsory
```
Why?

Reply:
@Thomas-Upfield I am also curious "why" because of the reasons I mentioned at yesterday's CIP meeting:
- It might be well part of a human dRep's platform _not_ to have any of these references, or even to disavow such references.
- The dRep conception is currently aligned to human representatives but could be AIs, smart contracts aggregating other choices or behaviour, or other such abstracts soon enough... and these generally won't have references associated with human identity.

Reply:
I have changed this so that the only properties that are compulsory is the name property. This is the case because a person must have a name.

Reply:
@Thomas-Upfield I would agree with the qualification that it's compulsory because an _entity_ must have a name.  I have still seen no compelling argument or proposed requirement (cc @Ryun1) that a dRep must be a person.